docs/sections/contributor_guide/fork_pr_model.rst

```diff
@@ -1,4 +1,46 @@
 Fork and PR Model
 =================
 
-TBD
+Overview
+--------
+
+Contributions to the ``workflow-tools`` project are made via a `Fork <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks>`_ and `Pull Request (PR) <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests>`_ model. GitHub provides a thorough description of this contribution model in their `Contributing to projects` `Quickstart <https://docs.github.com/en/get-started/quickstart/contributing-to-projects>`_, but the steps, with respect to ``workflow-tools`` contributions, can be summarized as:
+
+#. `Fork <https://docs.github.com/en/get-started/quickstart/contributing-to-projects#forking-a-repository>`_ the ``workflow-tools`` `repository <https://github.com/ufs-community/workflow-tools>`_ into your personal GitHub account.
+#. `Clone <https://docs.github.com/en/get-started/quickstart/contributing-to-projects#cloning-a-fork>`_ your fork onto your development system.
+#. `Create a branch <https://docs.github.com/en/get-started/quickstart/contributing-to-projects#creating-a-branch-to-work-on>`_ in your clone for your changes.
+#. `Make, commit, and push changes <https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-and-pushing-changes>`_ in your clone / to your fork. (Refer to the :doc:`Developer Setup <developer_setup>` page for setting up a development shell, formatting and testing your code, etc.)
+#. When your work is complete, `create a pull request <https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-a-pull-request>`_ to merge your changes.
+
+For future contributions, you may delete and then re-create your fork, or configure the official ``workflow-tools`` repo as `remote repository <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-repository-for-a-fork>`_ on your clone and `sync upstream changes <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork>`_ to stay up-to-date with the official repo.
+
+Specifics for ``workflow-tools``
+--------------------------------
+
+When creating your PR, please follow these guidelines, specific to the ``workflow-tools`` project:
+
+* Ensure that your PR is targeting base repository ``ufs-community/workflow-tools`` and base branch ``develop``.
```

Comment:
```suggestion
* Ensure that your PR is targeting the base repository ``ufs-community/workflow-tools`` and the base branch ``develop``.
```

Reply:
I don't think that's correct. This usage parallels e.g.
> Present at the after-party were best actor Brendan Fraser and best supporting actor Ke Huy Quan.

One wouldn't write
> Present at the after-party were the best actor Brendan Fraser and the best supporting actor Ke Huy Quan.
because the "the" would imply that the adjectives are intrinsic properties of the nouns, that Brendan Fraser is THE BEST ACTOR, which... well, I haven't even seen _The Mummy_. But "best actor" just relates to ephemeral circumstances. Writing "the base branch `develop`" would likewise imply that `develop` is somehow THE base branch, but "base" is just an ephemeral condition of this transaction.

I think, alternatively,
> ...is targeting, as its base repository, `ufs-community/workflow-tools`...
also sounds right, as it doesn't ascribe "base" permanently to the repo and makes it clear that "base" is in relation to the PR -- but it's also longer and wordier.

Also sounds right: "the repository" and "the branch" without "base" at all, but I've included "base" here because it appears in the GitHub dialog and I'm trying to draw the reader to the right place on the screen.

I'm not sure how to justify why the "the" sounds so wrong to me here, but it does. Probably a linguist would know the right term for whatever rule is in play. I feel like it has something to do with the fact that the adjective is valid due to a relationship between two things, and the "the" makes it belong to one of the things instead.

Reply:
That's fine. It just sounded funny to me when reading the rendered doc. The more I read it, the more it sounds fine as-is.